include/glow/Graph/Graph.h

```diff
@@ -566,25 +566,25 @@ class Function final : public Named {
 
   BatchedReduceAddNode *createBatchedReduceAdd(llvm::StringRef name,
                                                NodeValue batch,
-                                               unsigned_t axis);
+                                               llvm::ArrayRef<unsigned_t> axes,
+                                               bool keepDims = false);
```

Comment:
I see there weren't doxygen comments here before, but would you mind adding them here too?  and to the next `createBatchedReduceAdd()` too? I think it should be almost identical to those for `BatchedReduceMean`.